src/util/CMakeLists.txt

```diff
@@ -36,8 +36,8 @@ target_link_libraries(
          clio_options
 )
 
-if (is_gcc)
-  # FIXME: needed on gcc for now
+if (is_gcc OR is_clang)
```

Comment:
Could it be something else? msvc exists but we don't support it.

Reply:
The other option we have for this `if` is `is_appleclang` which is the one that we try to dodge here. As for MSVC and Windows - I'd not bother adding any extra checks for it as we don't support Windows anyway. Wdyt?

Reply:
I think apple clang also requires this flag. About windows I agree that we don't need to be bothered.